### DISCUSSION TITLE
获取群组信息没有成员数之类的信息

### DISCUSSION THREAD

**Discussion by shumyun** [koishijs/koishi#352] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

群组信息是通过ID获取ID和名字，这样的信息基本无用吧。群组总人数和上限其实想调用的人应该有的吧（例如我^v^）
**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by NWYLZW** [comment]
gocq 好像不支持（摊手.png

**Comment by shumyun** [comment]
> gocq 好像不支持（摊手.png

我看他们官网有这个[API](https://docs.go-cqhttp.org/api/#%E8%8E%B7%E5%8F%96%E7%BE%A4%E4%BF%A1%E6%81%AF)

**Comment by NWYLZW** [comment]
刚刚翻了一下文档已经支持，我记错了
不过之前讨论过，因为其他的平台
例如 discord ，它的群成员数获取方式不太一样，处理起来会比较麻烦
所以暂时没做

**Comment by shumyun** [comment]
> 刚刚翻了一下文档已经支持，我记错了
> 不过之前讨论过，因为其他的平台
> 例如 discord ，它的群成员数获取方式不太一样，处理起来会比较麻烦
> 所以暂时没做

我要自己获取，有啥API接口吗

**Comment by shigma** [comment, marked as answer]
自己获取可用 [bot.$getGroupInfo()](https://koishi.js.org/api/adapter/onebot.html#bot-getgroupinfo-groupid-nocache)。

![image](https://user-images.githubusercontent.com/33423008/130213522-1cbc73ea-a6a4-4314-acf7-4ca39a3b8280.png)

**Comment by shumyun** [comment]
> 自己获取可用 [bot.$getGroupInfo()](https://koishi.js.org/api/adapter/onebot.html#bot-getgroupinfo-groupid-nocache)

了解，谢谢
